### ISSUE TITLE
Help getting relay to work

### ISSUE THREAD

**Issue by brennants2658** [Mixiaoxiao/Arduino-HomeKit-ESP8266#66]
I've spent hours upon hours trying to find out how to get the toggle in the home app to open/close my relay. I must be missing something obvious. What code should I be adding where to get this to work?

**Comment by shibbidydibbidy** [comment]
Just use simplest_led it is basically plug and play. Just set your PIN_LED to the pin you are using to control your relay.

**Comment by alperenunuvar** [comment]
@shibbidydibbidy is it possible more than one relay to add? how?

**Comment by shibbidydibbidy** [comment]
> @shibbidydibbidy is it possible to control multiple relay? how?

Yeah just connect both relays in parallel. Use the same PIN_LED for both.

**Comment by alperenunuvar** [comment]
Thank you very much for your reply, but if we use the same PIN_LED for both parallel relays, when I try to close one of them, are both of them closed? Could you explain it?

**Comment by alperenunuvar** [comment]
> > @shibbidydibbidy is it possible to control multiple relay? how?
> 
> Yeah just connect both relays in parallel. Use the same PIN_LED for both.

Thank you very much for your reply, but if we use the same PIN_LED for both parallel relays, when I try to close one of them, are both of them closed? Could you explain it?
